OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -94,6 +94,14 @@ void ActorRemoved(Actor a)
 
 		void ITick.Tick(Actor self)
 		{
+			if (Self.Owner.WinState != WinState.Undefined)
+			{
+				if (Powers.Count > 0)
+					Powers.Clear();
+
+				return;
```

Comment:
No need to return here - looping over an empty array is fine and lets us simplify down to
```csharp
// Dead players should never be able to use support powers
if (Self.Owner.WinState != WinState.Undefined && Powers.Count > 0)
	Powers.Clear();
```